fuel/modules/fuel/libraries/Fuel_parser.php

```diff
@@ -82,7 +82,6 @@ public function initialize($params = array())
 		}
 		$this->set_engine($this->fuel->config('parser_engine'));
 		$this->create_compile_dir();
-		return $this;
```

Comment:
I don't think any other `initialize()` method returns `$this`. The only use of `Fuel_parser::initialize` calls it and ignores the return value anyway!